docs/directives/write-to-json.md

```diff
@@ -0,0 +1,43 @@
+# Write to JSON
+
+WRITE-TO-JSON directive converts the record into JSON.
+
+## Syntax
+```
+  write-to-json <column>
```

Comment:
this in my opinion is not write-to-json, this is write-to-json-map or something. write-to-json on the example below should create a body = "{\"int\":1,\"string\":\"this is a string\"}"